### ISSUE TITLE
The table does not have the specified index: [my-index]

### ISSUE THREAD

**Issue by aireater** [aws-amplify/amplify-cli#2247]
**Describe the bug**
The project has an index on a table that was created outside of amplify using custom resolvers (pre-@key functionionality). When `amplify mock` is running the following error appears in the terminal window:

`The table does not have the specified index: accessKey-index`

How are we supposed to manage indexes that were created outside of amplify? Move them all to `@key`? 

**To Reproduce**
1. Create an index on DDB table 
2. Create the appropriate resolvers in CustomResources.json to user the index created in step 1
3. Run `amplify mock`
4. Access function that uses the index
5. See error

**Expected behavior**
Not sure what I expect. This is seems like a pre-@key gap that is now filled. Should I be replacing the custom resolvers with `@key`s?

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Firefox
 - Version 3.0.0

**Comment by kaustavghosh06** [comment]
@aireater Yes, you're right, when you run "mock" locally - the CLI doesn't have any context of your indexes created outside the CLI - from the console. CLI would nevertheless know about indexes generated using @key - so you can use that instead.
